vit-servicing-station-tests/src/tests/graphql.rs

```diff
@@ -0,0 +1,9 @@
+use crate::common::startup::quick_start;
+use assert_fs::TempDir;
+
+#[test]
+pub fn graphql_sanity_test() {
```

Comment:
I think it would be better to be explicit about what we are testing. In this case, If we get a proposal by id lets reflect it in the name somehow. Or am I missing the point of the test here? Is this gonna run all functionality in one test?